pallets/token-mux/src/mock.rs

```diff
@@ -0,0 +1,241 @@
+// Copyright 2023 Centrifuge Foundation (centrifuge.io).
+//
+// This file is part of the Centrifuge chain project.
+// Centrifuge is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version (see http://www.gnu.org/licenses).
+// Centrifuge is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+use cfg_mocks::pallet_mock_token_swaps;
+use cfg_types::{
+	orders::OrderInfo,
+	tokens::{CustomMetadata, LocalAssetId},
+};
+use frame_support::{
+	parameter_types,
+	traits::{ConstU16, ConstU32, ConstU64, GenesisBuild},
+	PalletId,
+};
+use orml_traits::{asset_registry::AssetMetadata, parameter_type_with_key};
+use sp_core::{crypto::AccountId32, H256};
+use sp_io::TestExternalities;
+use sp_runtime::{
+	testing::Header,
+	traits::{BlakeTwo256, IdentityLookup},
+	FixedU128,
+};
+
+use crate::{
+	pallet as pallet_token_mux,
+	tests::{
```

Comment:
NIT. Right now, `mock` uses the `tests` module, and the `tests` module uses `mock`. If the mock uses these constants to initialize the testing state, could it then own them?

Reply:
I agree. I first had them in `mock` and then wanted to follow your example. But its cleaner to keep them in mock. In general, I prefer having them there.

Reply:
Yeah, I always have the dichotomy of where to put them... depends a lot if the mock uses them directly or not.

Reply:
Moved all constants back to the mock because that uses all of them